README.md

```diff
@@ -16,11 +16,109 @@ var backlight = require('rpi-backlight');
 
 backlight.powerOn();
 backlight.powerOff();
+// 1 = Screen is OFF
+// 0 = Screen is ON
+backlight.getActualPowerValue();
+
+backlight.setMaxBrightness();
+backlight.getActualBrightness();
+// The screen goes Off at <= 9 brightness value
+backlight.setBrightnessValue(value);
```

Comment:
This would be a slightly nicer api as `.getBrightness()` and `setBrightness(value)`, and `getBrightness` should probably return a number, not a string.